aksel.nav.no/website/components/website-modules/examples/__parts/ApplicationPictogram.tsx

```diff
@@ -1,14 +1,17 @@
 import React from "react";
 
-const ApplicationPictogram = () => (
+const ApplicationPictogram = ({
+  ...restProps
+}: Omit<React.SVGProps<SVGSVGElement>, "ref">) => (
   <svg
     xmlns="http://www.w3.org/2000/svg"
-    width="72"
-    height="72"
+    width="96"
+    height="96"
     viewBox="0 0 72 72"
     fill="none"
     style={{ flexShrink: 0 }}
     aria-hidden
+    {...restProps}
   >
```

Comment:
I noticed that this could be simplified

```suggestion
const ApplicationPictogram = (props: React.SVGProps<SVGSVGElement>) => (
  <svg
    xmlns="http://www.w3.org/2000/svg"
    width="96"
    height="96"
    viewBox="0 0 72 72"
    fill="none"
    style={{ flexShrink: 0 }}
    aria-hidden
    {...props}
  >
```